src/Tribe/Main.php

```diff
@@ -3828,17 +3836,25 @@ public function isVenue( $postId = null ) {
 		 * @return bool Is it an organizer?
 		 */
 		public function isOrganizer( $postId = null ) {
+			static $is_organizer = [];
+
 			if ( $postId === null || ! is_numeric( $postId ) ) {
 				global $post;
 				if ( isset( $post->ID ) ) {
 					$postId = $post->ID;
 				}
 			}
+
+			// Return if we've already fetched this info.
+			if ( isset( $is_organizer[ $postId ] ) ) {
+				return $is_organizer[ $postId ];
```

Comment:
:no_entry_sign: Variable "$postId" is not in valid snake_case format, try "$post_id"

<sub>hash: b161023b24b3e3e69560074b6d328b8d</sub>

Reply:
Ignoring.